msteams-platform/bots/how-to/bot-configuration-experience.md

```diff
@@ -13,8 +13,22 @@ ms.localizationpriority: high
 >
 > Bot configuration experience is supported in channel or group chat scopes only.
 
-When you're creating a bot for Teams, it’s important to make sure users have a smooth experience and provision control to personalize their bots in places like Microsoft Teams. The ability to configure bots post-installation within Teams allows developers to fine-tune bot functionality according to user requirements even after deployment. You can use methods from the Microsoft Bot Framework SDK to implement bot configuration features that enhance user engagement and productivity. 
- 
+Bot configuration allows developers to enhance user engagement and personalize bot experiences within platforms like Microsoft Teams. By enabling users to configure bot settings post-installation, developers can ensure a smoother onboarding process and tailor bot functionality to meet specific user needs.
+
+## Why Bot Configuration?
+
+Imagine a scenario where a user installs a bot in their Teams but finds the default settings inadequate for their workflow. Without bot configuration, users might feel limited by the bot's predefined behavior, leading to disengagement and decreased usage. However, by implementing bot configuration, developers empower users to tailor bot settings according to their preferences and requirements.
```

Comment:
@surbhigupta I know this is still in progress and not ready for review, but to short-circuit a round of review, this section is very important and needs to be refined. The current text feels a bit limp and reads as corporate-speak.

Don't tell me that this "empower[s] users to tailor bot settings according to their preferences and requirements", or that it "ensures that interactions align with their workflow and preferences"-- **show me an example of how an app can do that with this feature**.